render/detailed_node_test.go

```diff
@@ -1,4 +1,4 @@
-package report_test
+package render
```

Comment:
Right now the tests reference renderTopology, so need to be in the same namespace.

In a future PR I've merged renderTopology into Map.Render, so the tests can be moved out.  I was trying to keep changes to a minimum in this PR.